services/src/main/java/org/exoplatform/addons/gamification/service/setting/zone/impl/ZoneRegistryImpl.java

```diff
@@ -39,15 +46,41 @@ public boolean remove(ZoneConfig zone) {
     @Override
     public void start() {
 
-        for (ZoneConfig zone : zoneMap.values()) {
-            CommonsUtils.getService(SettingService.class).set(Context.GLOBAL, Scope.GLOBAL.id(null), (GAMIFICATION_PREFERENCES + zone.getZoneName()), SettingValue.create(zone.getZoneName()));
+        try {
+            // Processing registered domains
 
+            for (ZoneConfig domain : zoneMap.values()) {
```

Comment:
Is there a reason to do this check at each PLF startup ?

Reply:
Domains are created from configuration files that can be added from any extension, so we need to get domains from conf and create new ones every startup

Reply:
ok